math-magians/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
```

Comment:
- [ ] Your files should be contained on the root path, however, they are contained in the `math-magians` folder. Kindly move your files from the math-magians to the root path. This can aid navigation and future deployments.

  ![butler-readme](https://user-images.githubusercontent.com/36542986/153918132-218d05ab-d6b6-4b34-82d2-d416e211b718.jpg)

- [ ] While your README on the root path contains a descriptive paragraph however it is still missing more sections as contained in the Microverse template. Kindly format your entire README here using this [template](https://github.com/microverseinc/readme-template) and replace the initial one with the new one.
- [ ] In your README, remember to include steps for getting the app locally.
- [ ] Also remember to include `npm install` and `npm start` as necessary commands for installing packages and running the app.